pwa/src/layout/Layout.tsx

```diff
@@ -1,18 +1,41 @@
+import * as React from "react";
 import { Document, Page, PageContent } from "@nl-design-system-unstable/example-next.js/src/components/utrecht";
+import { isLoggedIn } from "../services/auth";
+import { APIProvider } from "../apiService/apiContext";
+import APIService from "../apiService/apiService";
+import Login from "../pages/login";
+import "../styling/index.css";
 import Footer from "./../components/footer/Footer";
 
 const Layout: React.FC = ({ children }) => {
-  return (
-    <>
-      <title>Skeleton Application</title>
+  const [API, setAPI] = React.useState<APIService | null>(null);
+
+  React.useEffect(() => {
+    if (!isLoggedIn()) {
+      setAPI(null);
+      return;
+    }
 
-      <Document>
-        <Page className="Page">
-          <PageContent className="PageContent">{children}</PageContent>
-          <Footer />
-        </Page>
-      </Document>
-    </>
+    const jwt = sessionStorage.getItem("jwt");
+    !API && jwt && setAPI(new APIService(jwt));
+  }, [API, isLoggedIn()]);
+
+  return (
+    <Document>
+      <Page className="Page">
+        <PageContent className="PageContent">
+          {API ? (
+            <APIProvider value={API}>
+              <title>Skeleton Application</title>
+              {children}
+            </APIProvider>
+          ) : (
+            <Login />
+          )}
+        </PageContent>
+        <Footer />
```

Comment:
Because it's very likely that the `Footer` will be containing internal links that require a user to be authenticated; I suggest moving this to the `APIProvider` block.

Note: this will break the positioning of the Footer, so please update that as well.